justfile

```diff
@@ -17,7 +17,11 @@ alias w := watch
 
 # Run tests (backend & frontend)
 test:
-    cargo watch -- cargo leptos test
+    cargo-test
```

Comment:
Why was `cargo watch -- cargo leptos test` replaced?

Reply:
Oh, I just realized that tests were broken on `main`. I just fixed it in 855e964924bf94ea78aad04b16add08512db7e34 - you want to merge that.